install/zoraxy-install.sh

```diff
@@ -20,20 +20,11 @@ $STD apt-get install -y mc
 $STD apt-get install -y git
 msg_ok "Installed Dependencies"
 
-msg_info "Installing Golang"
-set +o pipefail
-RELEASE=$(curl -s https://go.dev/dl/ | grep -o "go.*\linux-amd64.tar.gz" | head -n 1)
-wget -q https://golang.org/dl/$RELEASE
-$STD tar -xzf $RELEASE -C /usr/local
-$STD ln -s /usr/local/go/bin/go /usr/local/bin/go
-set -o pipefail
-msg_ok "Installed Golang"
-
-msg_info "Installing Zoraxy (Patience)"
-$STD git clone https://github.com/tobychui/zoraxy /opt/zoraxy
-cd /opt/zoraxy/src
-$STD go mod tidy
-$STD go build
+RELEASE="$(curl -s https://api.github.com/repos/tobychui/zoraxy/releases | grep -oP '"tag_name":\s*"\K[\d.]+?(?=")' | sort -V | tail -n1)"
+msg_info "Installing Zoraxy v${RELEASE}"
+wget -q "https://github.com/tobychui/zoraxy/releases/download/${RELEASE}/zoraxy_linux_amd64"
+install zoraxy_linux_amd64 /usr/bin/zoraxy
+echo "${RELEASE}" > "/opt/${APPLICATION}_version.txt"
```

Comment:
??

Reply:
Again, this was taken from the `traefik-install` script. 
The file is used during the update process to check installed vs available versions:
https://github.com/tteck/Proxmox/pull/3963/files#diff-0f39facf72c262912c85eac14c5b8848d6356fe9991710f7a1c11d5eb64a363eR60